test/server/graph/mutations/removeCommentTag.js

```diff
@@ -34,34 +34,53 @@ describe('graph.mutations.removeCommentTag', () => {
     const context = new Context({user});
 
     // add a tag first
-    await CommentsService.addTag(comment.id, 'BEST');
+    await CommentsService.addTag(comment.id, 'BEST', user.id);
     const response = await graphql(schema, query, {}, context, {id: comment.id, tag: 'BEST'});
     if (response.errors && response.errors.length) {
       console.error(response.errors);
     }
     expect(response.errors).to.be.empty;
     expect(response.data.removeCommentTag.errors).to.be.null;
-    expect(response.data.removeCommentTag.comment.tags).to.deep.equal([]);
+
+    CommentsService.findById(response.data.removeCommentTag.comment.id)
+    .then(({tags}) => {
+      expect(tags).to.deep.equal([]);
+    });
+
   });
 
   describe('users who cant remove tags', () => {
-    Object.entries({
-      'anonymous': undefined,
-      'regular commenter': new UserModel({}),
-      'banned moderator': new UserModel({roles: ['MODERATOR'], status: 'BANNED'})
-    }).forEach(([ userDescription, user ]) => {
-      it(userDescription, async function () {
-        const context = new Context({user});
 
-        // add a tag first
-        await CommentsService.addTag(comment.id, 'BEST');
-        const response = await graphql(schema, query, {}, context, {id: comment.id, tag: 'BEST'});
-        if (response.errors && response.errors.length) {
-          console.error(response.errors);
+    // allow the tag in the settings
+    SettingModel.findOneAndUpdate({id: 1}, {
```

Comment:
All database operations should be in a `beforeEach/before/it` block, otherwise it's not scoped properly for tests.